wrapper_malconf.py

```diff
@@ -6,17 +6,27 @@
 sys.path.insert(0, RAT_DECODERS_PATH)
 from malwareconfig import fileparser
 from malwareconfig.modules import __decoders__, __preprocessors__
+import logging
```

Comment:
Suggestion: might be a good idea to use the log.init_logging from the AL base if not running as cli?
Just to be sure we're not deviating from the logging standard without knowing

https://github.com/CybercentreCanada/assemblyline-base/blob/fc5f8216e7fa59d9421ff626927d9602e5a3430c/assemblyline/common/log.py#L41

Reply:
Not sure this is the best way, as it is it grabs the service logger and that logger follows the logging standard. No need to duplicate it. Here's how the logs currently look.
![image](https://user-images.githubusercontent.com/62077906/112350296-b0460680-8c9f-11eb-9b0b-1296e853b391.png)
@cccs-kevin thoughts

Reply:
I haven't used `init_logging` before but it feels like the more natural fit

Reply:
Can you try running your test with `log_level: WARNING` in your config.yml? 
The expectation is no info logs should show up as in your screenshot if that config is set to something other than INFO which is the default. That's mostly what I'm looking for in this case.